templates/cf.yml

```diff
@@ -631,6 +654,9 @@ properties:
     blacklisted_syslog_ranges: ~
     unmarshaller_count: (( merge || 5 ))
     port: (( merge || 4443 ))
+    etcd:
+      client_cert: ~
+      client_key: ~
```

Comment:
That's fine with us. I just didn't want to make us dependent on the `etcd.client_cert` and `etcd.client_key` properties. We can create another etcd client cert and share it across the loggregator components if that is simpler.